E/EconPDEs/Compat.toml

```diff
@@ -1,13 +1,16 @@
-["0.2"]
+["0.2.0"]
+DataStructures = "0.2-0.15"
+
+["0.2.0-0.2.1"]
 ForwardDiff = "0.0-0.10"
 Interpolations = "0.0-0.11"
 NLsolve = "0.1-4.0"
 Reexport = "0.0-0.2"
 julia = "0.7-1.3"
 
-["0.2.0"]
-DataStructures = "0.2-0.15"
-
 ["0.2.1"]
 DifferentialEquations = "0.0-6.3"
 OrderedCollections = "0.1-1.1"
+
+["0.2.2"]
+julia = "0.7-1.1"
```

Comment:
This is wrong, see https://github.com/JuliaComputing/Registrator.jl/issues/103. Linking it with that issue so we can correct it later.